res/openvpn-gui-res-no.rc

```diff
@@ -272,9 +272,9 @@ BEGIN
     IDS_NFO_STATE_FAILED_RECONN "Status: Kunne ikke koble til på nytt"
     IDS_NFO_RECONN_FAILED "Tilkobling til %s feilet."
     IDS_NFO_STATE_SUSPENDED "Status: Hvilemodus"
-    IDS_ERR_READ_STDOUT_PIPE "Feil under lesning fra OpenVPN StdOut pipe."
-    IDS_ERR_CREATE_EDIT_LOGWINDOW "Opprettelse av RichEdit LogWindow feilet!"
-    IDS_ERR_SET_SIZE "Set Size feilet!"
+    IDS_ERR_READ_STDOUT_PIPE "Feil under lesning fra OpenVPN standard ut-kanal."
```

Comment:
I think "pipe" is too technical in messages meant for the end-user. Instead use something like "OpenVPN console output" in place of "OpenVPN stdout pipe"?